packages/rich-text/src/component/index.native.js

```diff
@@ -770,25 +771,28 @@ export class RichText extends Component {
 			style,
 			__unstableIsSelected: isSelected,
 			children,
+			useStyle,
```

Comment:
I don't think it's a good idea to prefix the prop name with `use` as this confuses it with React hooks.

Reply:
Agreed here. cc @etoledom

Reply:
Something like `extendStyles()` or `extendStylesToTheme()` maybe?

Reply:
I agree too. There already were some issues with linting.